google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -1114,6 +1114,37 @@ public boolean deleteDefaultAcl(Entity entity) {
     return storage.deleteDefaultAcl(getName(), entity);
   }
 
+  /**
+   * Delete the lifecycle rules of this bucket.
+   *
+   * <p>Example of deleting the lifecycle rules of this bucket.
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * Bucket bucket =
+   *     storage.create(
+   *         BucketInfo.newBuilder(bucketName)
+   *             .setStorageClass(StorageClass.COLDLINE)
+   *             .setLocation("us-central1")
+   *             .setLifecycleRules(
+   *                 ImmutableList.of(
+   *                     new LifecycleRule(
+   *                         LifecycleAction.newDeleteAction(),
+   *                         LifecycleCondition.newBuilder().setAge(2).build())))
+   *             .build());
+   * boolean rulesDeleted = bucket.deleteLifecycleRules();
```

Comment:
How do I delete 1 lifecycle rule?

Reply:
@frankyn I think there is no possible way to delete or disable any 1 lifecycle rule please check https://cloud.google.com/storage/docs/managing-lifecycles#disable